### PR TITLE
expanding the plain (no framework) python example

### PR DESCRIPTION
**Pull request by tremlab**
adding further configuration details.

### CODE REVIEW

File: example/plain/README.md [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
 
-For more information on setting up Bugsnag in Python please consult the [official documentation](https://docs.bugsnag.com/platforms/python/other)
+1. Clone the repo and `cd` into this directory:
+  ```sh
+  git clone https://github.com/bugsnag/bugsnag-python.git
+  cd bugsnag-python/example/plain
+  ```
 
-## Running the example
+1. Before testing it, open up the `app.py` file and configure your API key.
```
Comment (Cawllec) [inline example/plain/README.md]: I think this line should have an example script with it, just something like:
```
bugsnag.configure(
    api_key = 'YOUR_API_KEY_HERE',
    ...
)
```

File: example/plain/README.md [hunk 2]
```diff
@@ -1,30 +1,34 @@
...
+1. Install dependencies
+  ```shell
+  pip install -r requirements.txt
+  ```
 
-After running the example head to the [Bugsnag dashboard](https://app.bugsnag.com) to see the generated notifications.
\ No newline at end of file
+1. Run the application in your terminal.
+  ```shell
+  python -i server.py
```
Comment (Cawllec) [inline example/plain/README.md]: This should be `app.py`

File: example/plain/app.py [hunk 3]
```diff
@@ -1,9 +1,119 @@
...
+    # automatically sends this manual notification every time the file is loaded.
+    bugsnag.notify(Exception('File loaded!'))
+
+    print("""
+    This app sends one notification to your Bugsnag dashboard on loading.
+
+    call this function to send more handled exceptions:
+    -  handle_zero_div()
+
+    call these functions to crash the app and send unhandled exceptions:
```
Comment (Cawllec) [inline example/plain/app.py]: Do we want more explanation, whether here or in the readme, about what this means in terms of callbacks?  Or at least to direct them to the correct point in the script?

Comment (tremlab) [reply example/plain/app.py]: I've gone back and forth on this -- It's explained clearly (I hope!) in the function description, and where the callback occurs.  I could add more details to this print statement, but then it gets cluttered.  I'd prefer not to add to the readme -- but of course, open to revision!

**Pull request merged**